File: papi/lista2/exercicio2.kt
package lista2

fun main(){
    var numero: Int = 0

    while (true) {
        try {
            print("Por favor informe um número: ")
            val input = readLine()
            numero = input!!.toInt()
            break
        } catch (e: Exception) {
            println("Erro na conversão de string para número")
        }
    }

    print("O numero informado foi: $numero")
}

File: papi/lista2/exercicio2.rb
print "Por favor informe um número: "
numero = gets.chomp
if numero == "0"
    puts "O número informado foi: 0"
else
    numero = numero.to_i
    if numero != 0
        puts "O número informado foi: #{numero}"
    else
        puts "Número inválido"
    end
end
